src/Microsoft.Identity.Web.DownstreamApi/DownstreamApi.cs

```diff
@@ -1,174 +1,176 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using System.IO;
-using System.Linq;
-using System.Net.Http;
-using System.Runtime.CompilerServices;
-using System.Security.Claims;
-using System.Text;
-using System.Text.Json;
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.

```

Comment:
Why did the formatting change?

Reply:
I am not sure.  I rebased now and I don't see the format change. Spacing is different for ppl using other editors or if you make and accept suggestions in GitHub. GH uses Linux newlines. VS use Windows.